docs/UserGuide.md

```diff
@@ -179,6 +179,72 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Creating a Tag: `addtag`
+
+Creates a new tag to Contax.
+
+Format: `addtag n/TAGNAME`
+
+* The `TAGNAME` parameter **must** be specified.
+* If the `TAGNAME` already exists (case-insensitive), the tag will not be created.
+
+Examples:
+* `addtag n/Potential Clients` Creates a tag named *Potential Clients* in the address book (if it does not exist).
+
+### Editing a Tag: `edittag`
+
+Edits an existing tag in Contax.
+
+Format: `edittag f/OLD_TAGNAME t/NEW_TAGNAME`
+
+* All parameters **must** be specified.
+* Edits a tag specified at `OLD_TAGNAME` and updates to the tag specified at `NEW_TAGNAME`.
+* `OLD_TAGNAME` and `NEW_TAGNAME` are case-insensitive.
+* If `OLD_TAGNAME` cannot be found, the command will not execute.
+* If `NEW_TAGNAME` already exists in Contax (case-insensitive), the command will not execute.
```

Comment:
I would suggest the command throw an error instead of failing silently.

Reply:
Noted. I have made this change in delete tag too